### DISCUSSION TITLE
SyntaxError: Unexpected token '<' - NextJS SVG doesnt work

### DISCUSSION THREAD

**Discussion by lucasReghiniM** [vercel/next.js#19560] [Help]
I'm using a personal package that have some components and svgs. I used to use this library with react, and everything gones fine. 
Yesterday i started with NextJS. I install the Next images plugin and inline-react-svg, but when i run the NPM RUN DEV, that error appears:

```
/home/****/Documents/workbench/next//****_site/node_modules//****_frontend_lib/dist/load~xBUfapxG.svg:2
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="margin: auto; background: none; display: block; shape-rendering: auto;" width="200px" height="200px" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid">
^

SyntaxError: Unexpected token '<'
    at wrapSafe (node:internal/modules/cjs/loader:1018:16)
    at Module._compile (node:internal/modules/cjs/loader:1066:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
    at Module.load (node:internal/modules/cjs/loader:967:32)
    at Function.Module._load (node:internal/modules/cjs/loader:807:14)
    at Module.require (node:internal/modules/cjs/loader:991:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/home//****/Documentos/workbench/next//****_site/node_modules//****_frontend_lib/dist/index.js:7:28)
    at Module._compile (node:internal/modules/cjs/loader:1102:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
```

**My next.config.js**
```
const withImages = require('next-images')
module.exports = withImages({
    esModule: true
});

```
**My babel.config.ts**

```
module.exports = {
    presets: ["next/babel"],
    plugins: [
        ["styled-components", { "ssr": true }],
        "inline-react-svg"
    ]
}
```
`My next-env.d.ts`
```
/// <reference types="next" />
/// <reference types="next/types/global" />
/// <reference types="next-images" />

```
**The problem component**
![image](https://user-images.githubusercontent.com/47827740/100355221-237c8c00-2fd0-11eb-9c3e-2ff6c62d8138.png)

Everything was supposed to work correctly with these plugins. Svgs that I import from a local folder work, but the error only appears when I installed the package with these svgs.

**Comment by jachinte** [comment]
@lucasReghiniM I'm having the same issue. This may be useful for you:

https://stackoverflow.com/a/60315083/738968

**Reply by reujwils** [reply]
@jachinte I want to thank you for sharing that thread!  I was having this exact issue and it did the trick.

**Reply by jachinte** [reply]
That's great, @reujwils!

**Reply by dreamawakening** [reply]
This didn't work for me, I'm not sure why. I'm still getting the "SyntaxError: Unexpected token '<'" error

**Comment by ShekharDhangar** [comment]
I have a private package that exports svg files and should be use these using `img` tags with src. How should I export these svgs? so that user using next  should not have any issues using it
